eth/utils/blake.py

```diff
@@ -0,0 +1,10 @@
+try:
+    # Python 3.6
+    from hashlib import blake2b
+except ImportError:
+    # Python 3.5
```

Comment:
I don't think we need to support 3.5, so maybe we can remove this and only focus on 3.6+ for beacon things.

Reply:
It's currently covered by `py35-core` tests. Do you think I should separate `beacon-utils` tests from `core`? Or we should work on another new branch instead of the `master` branch?

Reply:
How about we spin up a new CI run and relocate these tests to live under `tests/beacon` or maybe `tests/core-36` and we can merge once we drop 3.5 in core.

Reply:
Actually, ignore that, lets just mark all of those `3.6` only tests with a pytest skip

https://docs.pytest.org/en/latest/skipping.html#id1